src/Adapters/Auth/index.js

```diff
@@ -91,9 +119,17 @@ module.exports = function(authOptions = {}, enableAnonymousUsers = true) {
   const setEnableAnonymousUsers = function(enable) {
     _enableAnonymousUsers = enable;
   }
+
+  for (var prov in providers) {
+    const adapter = providers[prov];
+    const validationError = validateAdapter(adapter);
```

Comment:
I'm not sure why all this logic was added there, can you expand please?

Also having a method that returns an error is an anti-pattern, either it throws or returns false as validation didn't pass etc....

Reply:
You're absolutely right. I cannot explain why I put that validation there. There's no benefit from it whatsoever. Removed it.